espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityHelper.java

```diff
@@ -32,4 +56,42 @@ public static Activity getCurrentActivity() throws AppiumException {
 
         throw new AppiumException("Failed to get current Activity");
     }
+
+    private static String getFullyQualifiedActivityName(Instrumentation mInstrumentation, String appActivity) {
+        Context context = mInstrumentation.getTargetContext();
+        // If it's not fully qualified, make it fully qualified
+        return appActivity.startsWith(".")
+                ? context.getPackageName() + appActivity
+                : appActivity;
+    }
+
+    public static void startActivityViaInstruments(String appActivity, boolean waitForActivity) {
+        logger.info(String.format("Starting activity '%s'", appActivity));
+        Instrumentation mInstrumentation = InstrumentationRegistry.getInstrumentation();
+        Instrumentation.ActivityMonitor mSessionMonitor = mInstrumentation.addMonitor(appActivity, null, false);
```

Comment:
The monitor should use `fullyQualifiedAppActivity`.
In this case, if a user  use `appActivity: '.accessibility.AccessibilityNodeProviderActivity'`, the monitor cannot track `io.appium.android.apis.accessibility.AccessibilityNodeProviderActivity` correctly. (As a result, espresso server wait the status _STARTED_ even the activity already started by LifecycleMonitor